developers/weaviate/configuration/compression/multi-vectors.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Multi-vector compression
```

Comment:
Given that we already have pages for quantization techniques, would it make sense to scope this page around encoding techniques for multi-vector embeddings?

Reply:
You are right, maybe this is too much redundancy, I will only leave a comment/link that quantization is also an option for multi-vector embeddings.